src/components/pages/PendingApplications/ApplicationModal.js

```diff
@@ -159,139 +174,10 @@ const ApplicationModal = ({
-              </List.Item>
-              <List.Item>
-                <b>Email:</b> {currentApplication.email}
-              </List.Item>
-              <List.Item>
-                <b>Location:</b> {currentApplication.city},{' '}
-                {currentApplication.state} {currentApplication.country}
-              </List.Item>
-              <List.Item>
-                <b>Membership Criteria:</b>
-                <ul>
-                  {currentApplication.formerly_incarcerated === true ? (
-                    <li>Formerly Incarcerated</li>
-                  ) : null}
-                  {currentApplication.low_income === true ? (
-                    <li>Low Income</li>
-                  ) : null}
-                  {currentApplication.underrepresented_group === true ? (
-                    <li>Belongs to underrepresented group</li>
-                  ) : null}
-                </ul>
-              </List.Item>
-              <List.Item>
-                {' '}
-                <b>Convictions:</b>{' '}
-                {`${
-                  currentApplication.formerly_incarcerated === true
-                    ? currentApplication.convictions
-                    : 'none'
-                }`}
-              </List.Item>
-              <List.Item>
-                <b>Applicant needs help with:</b>{' '}
-                <ul>
-                  {currentApplication.industry_knowledge === true ? (
-                    <li>Industry Knowledge</li>
-                  ) : null}
-                  {currentApplication.pair_programming === true ? (
-                    <li>Pair Programming</li>
-                  ) : null}
-                  {currentApplication.job_help === true ? (
-                    <li>Job Help</li>
-                  ) : null}
-                </ul>
-              </List.Item>
-              <List.Item>
-                <b>Subject most interested in:</b> {currentApplication.subject}
-              </List.Item>
-              <List.Item>
-                <b>Other information:</b> {currentApplication.other_info}
-              </List.Item>
-              <List.Item>
-                <b>Submission Date:</b>{' '}
-                {currentApplication.created_at.slice(0, 10)}
-              </List.Item>
-              <List.Item>
-                <b>Application Status:</b> {currentApplication.validateStatus}
-              </List.Item>
-              <List.Item>
-                <b>Notes:</b> {currentApplication.application_notes}
-                <Button
-                  onClick={displayForm}
-                  hidden={!hideForm}
-                  block
-                  size="small"
-                  type="dashed"
-                  style={{ background: 'white', borderColor: '#1890ff' }}
-                >
-                  Edit Notes
-                </Button>
-              </List.Item>
-            </List>
+            <MenteeModal applicant={currentApplication} />
```

Comment:
Excellent job breaking down and moving more complex code into their own components. Makes the code much more readable